1.11/administering-clusters/licenses/license-api/index.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: layout.pug
+navigationTitle:  License API
+title: License API
+menuWeight: 3
+enterprise: true
+---
+
+You use the License API to manage DC/OS licenses. 
+
+# Routes
+
+Access to the License API is proxied through the Admin Router on each master node using the following route:
+
+```
+/system/v1/license/v1
+```
+
+To determine the URL of your cluster, see [Cluster Access](/1.11/api/access/).
+
+# Format
+
+The License API request and response bodies are formatted in JSON.
+
+Requests must include the accept header:
+
+```
+Accept: application/json
+```
+
+Responses include the content type header:
+
+```
+Content-Type: application/json
+```
```

Comment:
They can also be a tar file and no JSON, see: https://github.com/mesosphere/dcos-licensing/blob/master/docs/dcos-licensing-component-spec.yaml#L107

Reply:
I am not sure how to word this.

Reply:
Should this text appear as is, or is this a note?

Reply:
Content-Type: application/json

or

Content-Type: application/x-tar

Reply:
change made